CHANGELOG.md

```diff
@@ -1,5 +1,158 @@
 # CHANGELOG
 
+## 0.5.0 (2018-05-25)
+
+**IMPORTANT**: In Pouch 0.5.0 we have done many things that important to all users of Pouch:
```

Comment:
Please use `PouchContainer` here, since it is the brand we use now. If the part below has the same issue, please correct them as well. Thanks a lot.

Reply:
roger !